obd_ontology/gen_demo_kg.py

```diff
@@ -12,10 +12,11 @@
         "Plusleitung der Heizung der Lambdasonde",
         "Plusleitung der Lambdasonde",
         "Masseleitung der Lambdasonde",
-        "Signalleitung (Druck) des Saugrohrdrucksensors",
+        "Versorgungsspannung des Saugrohrdrucksensors",
         "Signalleitung (Temperatur) des Saugrohrdrucksensors",
         "Masseleitung des Saugrohrdrucksensors",
-        "Versorgungsspannung des Saugrohrdrucksensors"
+        "Signalleitung (Druck) des Saugrohrdrucksensors"
+
     ]
```

Comment:
I don't see why this is necessary, can you elaborate?

Reply:
I see, good catch. What the query returns is non-deterministic, though, so it could be wrong again with the next KG.